### PR TITLE
[Feature] Add support for Webhooks in Pay UI component

### PR DESCRIPTION
**Pull request by jnsdls**
CLOSES: DASH-88, DASH-87, DASH-85, DASH-86

---

## PR-Codex overview
This PR enhances the Pay UI component by adding a new "Webhooks" tab with functionality to create, view, and delete webhooks.

### Detailed summary
- Added `WebhooksPage` component with webhook management features
- Imported multiple UI components and hooks
- Implemented webhook creation, display, and deletion functionality

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by jnsdls** [comment]
* **#3810** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/3810?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jnsdls and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by jnsdls** [comment]
waiting for some API updates

File: apps/dashboard/src/@/components/ui/form.tsx [hunk 1]
```diff
@@ -112,12 +112,18 @@ const RequiredFormLabel = React.forwardRef<
     <ToolTipLabel label="Required">
       <Label
         ref={ref}
-        className={cn(error && "text-destructive", "relative", className)}
+        className={cn(
+          error && "text-destructive-foreground",
```
Comment (MananTank) [inline apps/dashboard/src/@/components/ui/form.tsx:116]: We should use `text-destructive-text` here instead ( I have a PR open that does this change - https://github.com/thirdweb-dev/js/pull/3829 )

File: apps/dashboard/src/@/components/ui/tabs.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export function TabLinks(props: {
         {/* Active line */}
         <div
           ref={lineRef}
-          className="absolute left-0 bottom-0 z-10 h-[2px] bg-foreground rounded-lg fade-in-0 animate-in"
```
Comment (MananTank) [inline apps/dashboard/src/@/components/ui/tabs.tsx:53]: Good catch, also added in https://github.com/thirdweb-dev/js/pull/3829/files

Comment (MananTank) [reply apps/dashboard/src/@/components/ui/tabs.tsx:53]: For some reason - vercel is not including this change in build and I'm also not getting it locally and there's this warning from graphite 

image

Comment (MananTank) [reply apps/dashboard/src/@/components/ui/tabs.tsx:53]: I did `gt track` and that fixed this warning, bu still not getting this particular change locally

Comment (MananTank) [reply apps/dashboard/src/@/components/ui/tabs.tsx:53]: looks like same in happening in vercel deployment, it is not including this change

**Review by MananTank — APPROVED** [review]

**Pull request merged**